library/debug.c

```diff
@@ -112,61 +112,72 @@ void mbedtls_debug_print_ret(const mbedtls_ssl_context *ssl, int level,
     debug_send_line(ssl, level, file, line, str);
 }
 
-void mbedtls_debug_print_buf(const mbedtls_ssl_context *ssl, int level,
-                             const char *file, int line, const char *text,
-                             const unsigned char *buf, size_t len)
+#define MBEDTLS_DEBUG_PRINT_BUF_NO_TEXT       0
```

Comment:
Question: I don't understand the motivation for this. Not saying it's wrong, actually I'm sure you had a good reason, it's just not apparent to me at the moment. Can you clarify?